gradle.properties

```
@@ -15,6 +15,7 @@ githubRepo = merequester
 
 # mod dependencies
 aeVersion = 19.2.7
+ae2wtlibVersion = 19.2.3
```

I found and fixed some bugs in ae2wtlib while testing this
```suggestion
ae2wtlibVersion = 19.2.5
```